api/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     implementation "net.javacrumbs.shedlock:shedlock-spring:2.4.0"
     implementation "net.javacrumbs.shedlock:shedlock-provider-jdbc-template:2.4.0"
     implementation "com.vladmihalcea:hibernate-types-52:2.4.3"
-    implementation "org.javers:javers-spring-boot-starter-sql:5.5.0:"
+    implementation "org.javers:javers-spring-boot-starter-sql:5.7.7:"
```

Comment:
An unrelated change. I got confused by the order of changes in Javers so updated it to while I was figuring it out. I decided it did no harm to be using the latest version.